### DISCUSSION TITLE
Need new release of docker image

### DISCUSSION THREAD

**Discussion by nottledim** [Koenkk/zigbee2mqtt#22218] [General]
The latest docker release comprises z2m= v1.36.1 and herdsman converters=19.11.2.  I think there ave been updates to herdsman for the new Aqara Ceiling Light T1M in the last few days.  Is there any chance of updating the docker image so I can get advantage of them (and release myself from ZHA with a separate network).  Much appreciated.

**Comment by chris-1243** [comment]
You may use the `latest-dev` tag while pulling your image.

**Comment by nottledim** [comment]
Thanks; sadly just too late. I discovered the addon "edge" version a little while ago, so just getting that to work now, forsaking docker.

The lamp seems to work like ZHA.  The z2m gui for it not working fooled me for while.

Much appreciated.
